modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -22,42 +22,195 @@ To provision bare-metal hosts to the cluster by using the BMO, you must set the
+--
+
+* *DHCP configuration* `bmh.yaml`:
++
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-worker-<num>-bmc-secret <1>
+  namespace: openshift-machine-api
+type: Opaque
+data:
+  username: <base64_of_uid> <2>
+  password: <base64_of_pwd> <2>
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-worker-<num> <1>
+  namespace: openshift-machine-api
+spec:
+  online: True
+  bootMACAddress: <nic1_mac_address> <3>
+  bmc:
+    address: <protocol>://<bmc_url> <4>
+    credentialsName: openshift-worker-<num>-bmc-secret <1>
+    disableCertificateVerification: True <5>
+    username: <bmc_username> <6>
+    password: <bmc_password> <6>
+  rootDeviceHints:
+    deviceName: <root_device_hint> <7>
+  preprovisioningNetworkDataName: openshift-worker-<num>-network-config-secret <8>
+----
++
+<1> Replace `<num>` for the worker number of the bare metal node in the `name` fields, the `credentialsName` field, and the `preprovisioningNetworkDataName` field.
++
+<2> Replace `<base64_of_uid>` and `<base64_of_pwd>` with the base64 string of the user name and password.
++
+<3> Replace `<nic1_mac_address>` with the MAC address of the bare metal node's first NIC. See the "BMC addressing" section for additional BMC configuration options.
++
+<4> Replace `<protocol>` with the BMC protocol, such as IPMI, RedFish, or others. Replace `<bmc_url>` with the URL of the bare metal node's baseboard management controller.
++
+<5> To skip certificate validation, set `disableCertificateVerification` to true.
++
+<6> Replace `<bmc_username>` and `<bmc_password>` with the string of the BMC user name and password.
++
+<7> Optional: Replace `<root_device_hint>` with a device path if you specify a root device hint.
++
+<8> Optional: If you have configured the network interface for the newly created node, provide the network configuration secret name in the `preprovisioningNetworkDataName` of the BareMetalHost CR.
+
++
+[NOTE]
+====
+If the MAC address of an existing bare metal node matches the MAC address of a bare metal host that you are attempting to provision, then the Ironic installation will fail. If the host enrollment, inspection, cleaning, or other Ironic steps fail, the Bare Metal Operator retries the installation continuously. See "Diagnosing a host duplicate MAC address" for more information.
+====
 
+. Create the bare metal node:
++
+[source,terminal]
+----
+$ oc -n openshift-machine-api create -f bmh.yaml
+----
++
+.Example output
+[source,terminal]
+----
+secret/openshift-worker-<num>-network-config-secret created
+secret/openshift-worker-<num>-bmc-secret created
+baremetalhost.metal3.io/openshift-worker-<num> created
+----
++
+Where `<num>` will be the worker number.
+
+. Power up and inspect the bare metal node:
```

Comment:
IBM style guide suggests to not use `power up` `https://www.ibm.com/docs/en/ibm-style?topic=word-usage` 
Do not use. Use power on or turn on.

What do we inspect in the bare metal node? Any specific details??

Reply:
Inspect = boot the node and fetch hardware details, essentially.

Reply:
Updated As Suggested.